### ISSUE TITLE
Unable to initialize due to network cencorship in some countries

### ISSUE THREAD

**Issue by yeungkaho** [nextcloud/vm#508]
## Steps to reproduce
1. Be inside China or whatever stupid country that blocks google.com
2. try to initialize nextcloud

## Expected behaviour
everything works

## Actual behaviour
1. vm/static/test_connection.sh tests if google.com is reachable, which obviously is not in China
2. script thinks network is down, unable to continue

took me hour to realize this problem, plz fix

**Comment by enoch85** [comment]
OK, does github.com work?

**Comment by yeungkaho** [comment]
Thank you, didn't expect such a quick response! github.com will work, but I think www.baidu.com is better since in China basically everyone uses it to test internet reachability and Baidu is rock solid for this purpose.

**Comment by enoch85** [comment]
Yeah, but if Github is down it's not possible to `wget` the scripts anyway ;)

Will change this when I have some time left over.
